docs/docs/configuration/inactive-topics-detection.md

```diff
@@ -9,13 +9,18 @@ and configure other options in the Hermes Management configuration.
  detection.inactive-topics.enabled                           | enable inactive topics detection                                           | false         
  detection.inactive-topics.inactivity-threshold              | duration after which a topic is considered inactive and first notified     | 60d           
  detection.inactive-topics.next-notification-threshold       | duration after previous notification after which a topic is notified again | 14d           
- detection.inactive-topics.whitelisted-qualified-topic-names | list of qualified topic names that will not be notified event if inactive  | []            
- detection.inactive-topics.leader-election-zookeeper-dc      | datacenter of Zookeeper used for leader election for the detection job     | dc            
+ detection.inactive-topics.whitelisted-qualified-topic-names | list of qualified topic names that will not be notified event if inactive  | []
  detection.inactive-topics.cron                              | cron expression for the detection job                                      | 0 0 8 * * *   
  detection.inactive-topics.notifications-history-limit       | how many notification timestamps will be kept in history                   | 5             
 
 The detection job runs on a single instance of Hermes Management that is a
 leader based on the leader election Zookeeper instance.
 
+| Option                              | Description                                                                | Default Value |
+|-------------------------------------|----------------------------------------------------------------------------|---------------|
+| management.leadership.zookeeper-dc  | Specifies the datacenter of the Zookeeper instance used for leader election in the detection job | dc            |
```

Comment:
Looks like the formatting is off because of the too long Description.
Moreover, maybe description should be updated here? It still refers to detection job